screenpy/actions/see_all_of.py

```diff
@@ -37,26 +40,30 @@ class SeeAllOf:
         )
     """
 
+    tests: Tuple[T_T, ...]
+
     @classmethod
-    def the(cls, *tests: Tuple[Answerable, BaseResolution]) -> "SeeAllOf":
+    def the(cls: Type[SelfSeeAllOf], *tests: T_T) -> SelfSeeAllOf:
         """Supply any number of Question/value + Resolution tuples to test."""
         return cls(*tests)
 
-    def describe(self) -> str:
+    def describe(self: SelfSeeAllOf) -> str:
         """Describe the Action in present tense."""
         return f"See if all of {self.number_of_tests} tests pass."
 
     @beat("{} sees if all of the following {number_of_tests} tests pass:")
-    def perform_as(self, the_actor: Actor) -> None:
+    def perform_as(self: SelfSeeAllOf, the_actor: Actor) -> None:
         """Direct the Actor to make a series of observations."""
         for question, resolution in self.tests:
             the_actor.should(See.the(question, resolution))
 
-    def __init__(self, *tests: Tuple[Answerable, BaseResolution]) -> None:
-        if len(tests) < 2:
-            raise UnableToAct(
-                "Must supply 2 or more tests for SeeAllOf."
-                " Use See instead for single tests."
-            )
+    def __init__(self: SelfSeeAllOf, *tests: T_T) -> None:
+        for tup in tests:
+            if isinstance(tup, tuple):
+                if len(tup) != 2:
+                    raise UnableToAct("Tuple must contain two items")
```

Comment:
I think we could be more specific than "two items" here—"Tuple must contain a Question and a Resolution."?